src/box/iproto_features.c

```diff
@@ -54,6 +54,16 @@ mp_decode_iproto_features(const char **data, struct iproto_features *features)
 	return 0;
 }
 
+#define IPROTO_CONSTANT_MEMBER(s, v) {.name = #s, .value = v},
```

Comment:
Let's define `IPROTO_CONSTANT_MEMBER` in `src/box/iproto_constants.h`, near `struct iproto_constant`, and never undef it. It's name is unique so it shouldn't cause any name conflicts.

Reply:
Done. I failed to come up with a convinient comment for this define, so moved it in the header without any comment.